fairlearn/__init__.py

```diff
@@ -3,42 +3,19 @@
 
 """Tools for analyzing and mitigating disparity in Machine Learning models."""
 
+import atexit
+import logging
 import os
-import sys
 from .show_versions import show_versions  # noqa: F401
 
-
-# Finesse the version
-_FAIRLEARN_DEV_VERSION_ENV_VAR = "FAIRLEARN_DEV_VERSION"
-
-_base_version = "0.4.6"
-_dev_version = ""
-
-if _FAIRLEARN_DEV_VERSION_ENV_VAR in os.environ.keys():
-    dev_version_string = os.environ[_FAIRLEARN_DEV_VERSION_ENV_VAR]
-    try:
-        dev_version_value = int(dev_version_string)
-        if dev_version_value >= 0:
-            _dev_version = ".dev{0}".format(dev_version_value)
-        else:
-            msg = "Value of {0} was not greater than or equal to zero. Ignoring"
-            print(msg.format(_FAIRLEARN_DEV_VERSION_ENV_VAR), file=sys.stderr)
-    except ValueError:
-        msg = "Value of {0} in {1} did not parse to integer. Ignoring"
-        print(msg.format(dev_version_string, _FAIRLEARN_DEV_VERSION_ENV_VAR), file=sys.stderr)
-
-
 __name__ = "fairlearn"
-__version__ = "{0}{1}".format(_base_version, _dev_version)
-
+__version__ = "0.4.6.post0.dev0"
```

Comment:
post0 AND dev0?

Reply:
Yes. `post` because it's coming after `v0.4.6` and `dev` to stop it automatically superseding `v0.4.6` if this should get released by accident. I could do `v0.4.7.dev0` instead.

Reply:
I somehow expected to see 0.4.7.dev0
That's also consistent with sklearn https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/__init__.py

Reply:
I'd say the master branch is not a post release of the past main release, it's more of a dev release of the next release. If the last release is 0.4.6, then master should be 0.4.7.devN, since it's a development happening to release 0.4.7.

Reply:
Changed to 0.4.7.dev0 :-)